src/style.scss

```
@@ -57,16 +58,12 @@ $auro-inset-directions:'--squish';
   }
 }
 
-// handle focus-visible state
-:host(.focus-visible) {
-  .auro-button {
-    @include focus-visible;
+:focus-visible.auro-button {
```

Replace all of this with... 

```suggestion
:focus-visible,
.focus-visible {
  &.auro-button {
    @extend %focus-visible;
  }
}
```

---

Replace line 64 with the CSS from the previous mixin.

---

Unfortunately, this will break browsers that don't support :focus-visible. The Sass is compiled to 
```css
:focus-visible.auro-button,
.focus-visible.auro-button {
/* rules */
}
```
This rule will be [invalidated by browsers that don't support :focus-visible](https://developer.mozilla.org/en-US/docs/Web/CSS/Selector_list#selector_list_invalidation).

---

This is interesting to me. I always thought that the syntax means that the rule will be applied to both selectors. It's not an `and`, but an `or`. 

> This is because a single unsupported selector in a selector list invalidates the whole rule.

Look at me learning.

---

> This is interesting to me. I always thought that the syntax means that the rule will be applied to both selectors. It's not an and, but an or.

Yeah, it will be applied to both -- the issue is if a browser sees a selector it doesn't understand, it will throw the whole rule out.